contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -489,13 +489,19 @@
       },
       createNodesFromUploads(fileUploads) {
         fileUploads.forEach((file, index) => {
-          const title = file.original_filename
+          let title = file.original_filename
             .split('.')
             .slice(0, -1)
             .join('.');
+          if (title === undefined) {
```

Comment:
We should override the filename as the title if it is defined in the file metadata, so instead we should do:
```
if (file.metadata.title) {
```

Reply:
Sure